src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -73,4 +76,40 @@ object HTMLCleaner extends ConverterModule {
     el.html(el.html().replace("\u00a0", "")) // \u00a0 is the unicode representation of &nbsp;
   }
 
+  private def getIngressText(el: Element): Option[Element] = {
+    val firstSection = Option(el.select("body>section").first)
+    val ingress = firstSection.flatMap(section => Option(section.select(">p>strong").first))
+
+    ingress match {
+      case None => firstSection.flatMap(section => Option(section.select(">strong").first))
+      case x => x
```

Comment:
Ved nærmere ettertanke så tror jeg kanskje ikke _ vil binde til variabelen, slik at du kan returnere den på den måten. Personlig synes jeg Some(x) leser bedre. Men det er subjektiv matter of taste :)